dcos/package.py

```diff
@@ -1384,6 +1384,31 @@ def latest_version(self):
         pkg_versions.sort()
         return pkg_versions[-1]
 
+    def get_package_revision(self, package_version=None):
```

Comment:
No strong opinion on new names (what you mentioned was decided sounds fine to me), it's just clear the original names are causing problems.